experimental/devices/sn3218/doc.go

```diff
@@ -0,0 +1,11 @@
+// Copyright 2016 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+//
+// Package sn3218 controls a SN3218 LED driver with 18 LEDs over an i2c bus.
+// See cmd/sn3218/main.go for a usage example. Make sure to run Reset() after
+// New(), so that the state of the LEDs match the register of the chip.
+//
+// Datasheet
+// https://github.com/pimoroni/sn3218/blob/master/datasheets/sn3218-datasheet.pdf
```

Comment:
Please start `godoc -http=:6060` locally and browse to http://localhost:6060/pkg/periph.io/x/periph/experimental/devices/sn3218.

Then include an empty `//` in between the line Datasheet and the URL and refresh the page. You'll see how Datasheet becomes a title.

The right URL is http://www.si-en.com/uploadpdf/s2011517171720.pdf